### PR TITLE
Fixing coala issues

### PR DESCRIPTION
**Pull request by zmhassan**
## Description

Fixing failing bot tests.

### CODE REVIEW

**Comment by zmhassan** [comment]
Majority of the fixes coala recommended where all about adding comments which is good. I'll be adding more comments or can we skip the comments and just have it validate just pylint, pep8, flake8 the standard stuff.

**Comment by goern** [comment]
> [...] or can we skip the comments and just have it validate just pylint, pep8, flake8 the standard stuff.

You can, by removing the thoth-coala job from the pipeline. But I really dont see the point of replacing coala with pep8/flake8 as both will raise the same errors.

**Comment by goern** [comment]
@zmhassan this commit message "Removing pydocbear as its not necessary" is misleading, you are removing it because it is slowing down the process.

**Comment by zmhassan** [comment]
@goern Please merge this if this looks good to you ? @durandom  thoth-bot test passes for coala

**Comment by goern** [comment]
no, I dont merge on this repo, that is something who can do a review needs to do

**Comment by zmhassan** [comment]
@durandom Can you help us out here with a review and merge if this looks good?

File: .coafile [hunk 1]
```diff
@@ -7,7 +7,7 @@ max_lines_per_file = 1000
 max_line_length = 120
 
 [all.python]
-bears = PycodestyleBear, PyDocStyleBear
+bears = PycodestyleBear
```
Comment (durandom) [inline .coafile]: Why remove this bear?

Comment (zmhassan) [reply .coafile]: I will add it back in.

**Comment by zmhassan** [comment]
@goern @durandom So now its all passing with the documentation style rules. Can we finally merge this ?

File: .coafile (hunk 1 above)
Comment (zmhassan) [reply .coafile]: I've added it and it all passes coding style tests.

**Comment by zmhassan** [comment]
Also please remove stickler-ci since we are using thoth now so we can see this green.

**Comment by durandom** [comment]
Awesom. Now we have a celebration cola, ehrm, coala-🐻 

@zmhassan right, I've removed stickler. Shall we also remove travis?

**Comment by durandom** [comment]
just disabled travis... thoth FTW

**Pull request merged**